src/app/components/buscadorTurnosPrestaciones/turnos-prestaciones.component.ts

```diff
@@ -174,60 +174,28 @@ export class TurnosPrestacionesComponent implements OnInit {
                 }
             }
             if (tipo === 'prestaciones') {
-                if (value.value !== null) {
-                    this.parametros['prestacion'] = value.value.conceptId;
-                } else {
-                    this.parametros['prestacion'] = '';
-                }
+                value.value ? this.parametros['prestacion'] = value.value.conceptId : this.parametros['prestacion'] = '';
```

Comment:
```suggestion
                const data = value.values; // Este ponelo arriba de todo! (una sola vez)
                this.parametros['prestacion'] = data.conceptId || undefined;
```

Nunca uses el ternario así, la asignación va primero. De todas forma te sugiero más refactor. 
Sin mi refactor debería ir así:

```
this.parametros['prestacion'] = value.value ? value.value.conceptId : '';
```

Denominador común.

Reply:
El undifined evita que se mande le parámetro por la red. Pero en este caso puede provocar algún error en la API, porque ya esta acostumbrada a recibir una cadena vacía. Sino pincha dejalo, sino cambialo por '' por el momento

Reply:
No se porque use la ternaria asi, siempre hago la asignacion primero, cosas que pasan.
Al mandar undefined sigue pinchando asi que lo deje con la ternaria, bien usada.